### DISCUSSION TITLE
Is there any post exploitation modules that can loosen the defense of the system

### DISCUSSION THREAD

**Discussion by flamecopper** [rapid7/metasploit-framework#16243] [General]
For example, 

I am using the following, is there any of the same in metasploit?

**netsh firewall set opmode disable
netsh advfirewall firewall add rule name="ICMP Allow incoming V4 echo request" protocol=icmpv4:8,any dir=in action=allow
reg add "HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\Terminal Server" /v fDenyTSConnections /t REG_DWORD /d 0 /f**

**Comment by flamecopper** [comment]
In short,
 disable windows firewall
allow icmp
Allow RDP

**Reply by bcoles** [reply]
> disable windows firewall
> allow icmp
> Allow RDP

[modules/post/windows/manage/enable_rdp.rb](https://github.com/rapid7/metasploit-framework/blob/master/modules/post/windows/manage/enable_rdp.rb) does some of what you want.
